Modules/_cursesmodule.c

```diff
@@ -2627,13 +2669,24 @@ other A_* attributes.  pair_number() is the counterpart to this function.
 [clinic start generated code]*/
 
 static PyObject *
-_curses_color_pair_impl(PyObject *module, short color_number)
-/*[clinic end generated code: output=6a84cb6b29ecaf9a input=a9d3eb6f50e4dc12]*/
+_curses_color_pair_impl(PyObject *module, int color_number)
+/*[clinic end generated code: output=3fd752e8e24c93fb input=d4ed7238735f1647]*/
 {
     PyCursesInitialised;
     PyCursesInitialisedColor;
 
-    return  PyLong_FromLong(color_pair_to_attr(color_number));
+    if (color_number < _NCURSES_COLOR_VAL_MIN) {
+        PyErr_SetString(PyExc_OverflowError,
+                        "signed " _NCURSES_COLOR_VAL_TYPE_STR " is less than minimum");
+        return NULL;
+    }
+    else if (color_number > _NCURSES_COLOR_VAL_MAX) {
+        PyErr_SetString(PyExc_OverflowError,
+                        "signed " _NCURSES_COLOR_VAL_TYPE_STR "is greater than maximum");
```

Comment:
There is a space missing before the "is".

Reply:
Fixed.